### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by vaikas-google**

### CODE REVIEW

**Comment by duglin** [comment]
YUGE!
LGTM

**Comment by angarg12** [comment]
LGTM

**Comment by zrob** [comment]
LGTM

File: profile.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ the [Open Service Broker API specification](spec.md) SHALL take precedence.
   contain the credentials that the Application will use to communicate
   with the Service Instance.
 
-- *Application*: The software that will make use of, be bound to, a Service
+- *Application*: The software that uses a Service Instance via Service Binding.
```
Comment (pmorie) [inline profile.md]: This introduces a new typo because 'Instance' is still on the next line.

**Comment by duglin** [comment]
oh I thought he was going all hip-hop.  Except instead of "word!" it was "Instance!"

**Comment by vaikas-google** [comment]
Haha, sorry, I had not pushed the change that removed the extra instance of Instance :) PTAL

**Comment by avade** [comment]
LGTM

**Comment by mattmcneeney** [comment]
Needs re-L G T M 'ing @duglin @zrob @angarg12

**Comment by duglin** [comment]
LGTM

**Comment by angarg12** [comment]
LGTM

**Comment by pmorie** [comment]
LGTM

**Pull request merged**